kubernetes/client/apis/__init__.py

```diff
@@ -1,13 +0,0 @@
-from __future__ import absolute_import
```

Comment:
This file should have been re-created when you run apply-hotfixes.sh. Could you check if there was any error?

Reply:
I think it's fixed now.